### ISSUE TITLE
Contact Info markup improvements

### ISSUE THREAD

**Issue by inghamn** [City-of-Bloomington/drupal-theme-cob#276]
The contact information is visually grouped so that sighted users are able to understand the purpose of each link. However, screen reader users may not understand the purpose of each link. Links are grouped in description lists (<dl>), but a heading is missing that would provide screen reader users additional information that the items are grouped together.

Note that the links in question should also be in a single <dl>, not each in its own <dl>. Combining the definition terms (<dt>) and their related descriptions (<dd>) in the same <dl> will help make the grouping even more understandable to screen reader users.
